droidlet/perception/craftassist/heuristic_perception.py

```diff
@@ -374,7 +374,12 @@ def get_nearby_airtouching_blocks(
                                         idm[0], idm[1]
                                     )
                                 )
-    return blocktypes, shifted_c, tags
+        if tags:
+            shifted_c = [(l[0] + x - radius, l[1] + ymin, l[2] + z - radius) for l in c]
+            if len(shifted_c) > 0:
+                all_components.append(shifted_c)
+                all_tags.append(tags)
+    return blocktypes, all_components, all_tags
```

Comment:
can we add a test for this ?

Reply:
I know I know, I promise I will if we keep this more than a month. am thinking we are going to switch entirely to detr in a short enough timeframe that its not a good use of effort